$NetBSD: patch-Makefile,v 1.2 2020/07/02 21:40:50 nia Exp $

 - Remove MACOSX_DEPLOYMENT_TARGET as it will give problem on newer macOS
     and also targeting/building for current macOS version is enough

 - Honour LDFLAGS.

--- Makefile.orig	2020-04-23 01:16:42.000000000 +0000
+++ Makefile
@@ -12,7 +12,7 @@ OBJS= src/$T.o
 lib: src/lfs.so
 
 src/lfs.so: $(OBJS)
-	MACOSX_DEPLOYMENT_TARGET=$(MACOSX_DEPLOYMENT_TARGET); export MACOSX_DEPLOYMENT_TARGET; $(CC) $(LIB_OPTION) -o src/lfs.so $(OBJS)
+	$(CC) $(LDFLAGS) $(LIB_OPTION) -o src/lfs.so $(OBJS)
 
 test: lib
 	LUA_CPATH=./src/?.so lua tests/test.lua
